iptables/rules.go

```diff
@@ -92,15 +92,37 @@ func (c *Chain) RuleHashes(features *Features) []string {
 	// First hash the chain name so that identical rules in different chains will get different
 	// hashes.
 	s := sha256.New224()
-	s.Write([]byte(c.Name))
+	_, err := s.Write([]byte(c.Name))
+	if err != nil {
+		log.WithFields(log.Fields{
+			"chain": c.Name,
+		}).Debug("failed to write hash")
```

Comment:
Again, should probably panic here, this shouldn't fail and it'll break everything downstream if it does

Reply:
Fixed